tests/components/hue/test_config_flow.py

```diff
@@ -515,12 +515,9 @@ async def test_bridge_homekit(hass, aioclient_mock):
     result = await hass.config_entries.flow.async_init(
         const.DOMAIN,
         context={"source": config_entries.SOURCE_HOMEKIT},
-        data={
-            "host": "0.0.0.0",
-            "serial": "1234",
-            "manufacturerURL": config_flow.HUE_MANUFACTURERURL,
-            "properties": {"id": "aa:bb:cc:dd:ee:ff"},
-        },
+        data=zeroconf.ZeroconfServiceInfo(
+            host="0.0.0.0", properties={"id": "aa:bb:cc:dd:ee:ff"}
+        ),
```

Comment:
Note: `serial` and `manufacturerURL` are not available in `ZeroconfServiceInfo`, and are not used by the config_flow. I think it must have been a bad copy/paste from ssdp discovery.

Reply:
I think that is safe to assume indeed.